hack/build.sh

```diff
@@ -59,10 +59,12 @@ build-docker() {
         -e GOOS="${GOOS}"                                                 \
         -e GOARCH="${GOARCH}"                                             \
         -e CGO_ENABLED=1                                                  \
```

Comment:
Do we need to enable CGO here? In my mac os env it will produce some unrelated errors:
``` bash
net
# net
/usr/local/go/src/net/cgo_bsd.go:15:72: could not determine kind of name for C.AI_MASK
# os/user
/usr/local/go/src/os/user/getgrouplist_darwin.go: In function 'mygetgrouplist':
/usr/local/go/src/os/user/getgrouplist_darwin.go:16:11: warning: implicit declaration of function 'getgrouplist' [-Wimplicit-function-declaration]
  int rv = getgrouplist(user, (int) group, buf, ngroups);
           ^~~~~~~~~~~~
BUILD: dfget in /Users/yeya24/hub/gowork/Dragonfly/bin/dfget
net
# net
/usr/local/go/src/net/cgo_bsd.go:15:72: could not determine kind of name for C.AI_MASK
# os/user
/usr/local/go/src/os/user/getgrouplist_darwin.go: In function 'mygetgrouplist':
/usr/local/go/src/os/user/getgrouplist_darwin.go:16:11: warning: implicit declaration of function 'getgrouplist' [-Wimplicit-function-declaration]
  int rv = getgrouplist(user, (int) group, buf, ngroups);
           ^~~~~~~~~~~~
```